tasks/offlinePackagingTasks.ts

```diff
@@ -24,13 +24,13 @@ import { getAbsolutePathPackagesToInstall } from '../src/packageManager/getAbsol
 import { isValidDownload } from '../src/packageManager/isValidDownload';
 
 export const offlinePackages = [
-    { platformInfo: new PlatformInformation('win32', 'x86_64'), id: "win32-x64" },
-    { platformInfo: new PlatformInformation('win32', 'x86'), id: "win32-ia32" },
-    { platformInfo: new PlatformInformation('win32', 'arm64'), id: "win32-arm64" },
-    { platformInfo: new PlatformInformation('linux', 'x86_64'), id: "linux-x64" },
-    { platformInfo: new PlatformInformation('linux', 'arm64'), id: "linux-arm64" },
-    { platformInfo: new PlatformInformation('darwin', 'x86_64'), id: "darwin-x64" },
-    { platformInfo: new PlatformInformation('darwin', 'arm64'), id: "darwin-arm64" },
+    { platformInfo: new PlatformInformation('win32', 'x86_64'), id: "win32-x64", isFramework: true },
```

Comment:
Ensure we are only packaging the Full Framework version of O#. Packaging both in addition to our debugger and razor lsp may push our size over the marketplace limit. We should transition in a couple releases to packaging the SDK build of O# and enabling it by default.